docs/shop_administration/shop_administration.md

```diff
@@ -11,7 +11,7 @@ Depending on the chosen data provider and whether an ERP is connected, some tabs
 |[Cockpit](cockpit.md)|Basic statistics|
 |[Order Management](manage_orders.md)|Basic order management|
 |[Components](components.md)|A folder where [translations](translations.md) are stored. Can provide additional content which is used by the shop for project-specific information which is not displayed directly (as a full page) in the shop.|
-|[Configuration Settings](configuration_settings.md)|Basic shop configuration|
+|Configuration Settings|Basic shop configuration|
```

Comment:
I am thinking whether the table should not be followed by a note, somewhere along the lines of:
"Configuration settings are not user-editable. They have been described in detail in the development guide."
(and possibly a link).
Another thought - what category of users can see the "Configuration Settings" tab - is it visible to users other than Administrators, who are more likely to read the dev guide anyway? If so - would it make sense to remove the mention of the tab completely?

Reply:
The visibility depends on the permission setup of individual installations, so we can't define you will and who will not be able to see the tab.
The problem with "user-editable" is that they *are* user-editable, but in many cases a regular user should not use them. TBD when revising the whole user guide (soon (TM))

Reply:
In perso I have encountered a similar situation, where certainsettings are not visible to people without the "Edit" policy. I decided to state it clearly: "If you have this type of permissions, you can do this, and if you have that permissions you can do that".